tldw_Server_API/app/core/Writing/manuscript_analysis.py

```
@@ -157,13 +157,68 @@ def _extract_content(response: Any) -> str:
+                    extracted = _extract_content_block(content)
+                    if extracted:
+                        return extracted
+        content = response.get("content", "")
+        extracted = _extract_content_block(content)
+        if extracted:
+            return extracted
+        return ""
+    return _extract_content_block(response) or str(response)
+
+
+def _extract_content_block(content: Any) -> str:
```

Ruff (0.15.9)

[warning] Too many return statements (12 > 6)

(PLR0911)